internal/connector/test/integration/features/connector-multitenancy-api.feature

```diff
@@ -565,6 +639,10 @@ Feature: connector namespaces API
              {
                "key": "connector_mgmt.bf2.org/profile",
                "value": "default-profile"
+             },
+             {
+               "key": "connector_mgmt.bf2.org/connector_deployments",
+               "value": "0"
```

Comment:
I think we should avoid exposing such details to the users as this annotation is an internal one.

Reply:
Updated to reuse the status.connectors_deployed API property.